.eslintrc

```diff
@@ -699,21 +700,24 @@ rules:
     - 2
     - 3
 
-  # Enforce a maximum number of parameters in function definitions.
+  # Limits number of parameters that can be used in the function declaration.
   max-params:
     - 1
     - 5
 
+  # Enforce a maximum number of statements allowed per line.
+  max-statements-per-line:
+    - 1
+    -
+      max: 2
```

Comment:
Ahh, good eye. I accidentally carried that over from cfgov-refresh. So when it's set to `1`, lines like [this one](https://github.com/cfpb/cfgov-refresh/blob/flapjack/test/browser_tests/page_objects/page_wagtail_templates.js#L10) won't pass because both the whole line and the function body are statements, so it's two on one line. For short snippets like that `1` seemed too restrictive. Happy to update it back to `1` here though if you feel differently.